src/server/utils/oneAppDevCdn.js

```diff
@@ -0,0 +1,208 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
this file doesn't have test coverage, so a PR check is failing
https://github.com/americanexpress/one-app/actions/runs/6383698537/job/17324848213?pr=1125